.github/workflows/check_cirrus_cron.yml

```diff
@@ -11,8 +11,10 @@ name: check_cirrus_cron
 on:
     # Note: This only applies to the default branch.
     schedule:
-        # Assume cirrus cron jobs runs at least once per day
-        - cron:  '59 23 * * *'
+        # N/B: This should coorespond to a period slightly after
```

Comment:
Nit: Coors brewing has nothing to do with this PR :^)

Reply:
oof...I blame an oversensitive "o" key switch :grin: